docs/manual/contributing/_index.de.md

```diff
@@ -0,0 +1,257 @@
+
+![Fork this repository](/de/contributing/images/de/fork-this-repository.png?classes=shadow)
+
+
+### Inhalt bearbeiten
+
+Bearbeite den Inhalt, beschreibe deine Änderungen und klicke auf die Schaltfläche »Propose file change«.
+
+![Bearbeite den Inhalt](/de/contributing/images/de/inhalt-bearbeiten.png?classes=shadow)
+
+
+### Branch und Commit erzeugen
+
+GitHub wird nun einen Branch und einen Commit für deine Änderungen erzeugen ausserdem wird eine Vorschau deiner 
+Änderungen angezeigt:
+
+![Branch und Commit erzeugen](/de/contributing/images/de/branch-und-commit-erzeugen.png?classes=shadow)
+
+Wenn alles korrekt ist, klicke auf die Schaltfläche »Create pull request«.
+
+
+### Pull-Request erzeugen
+
+Auf der nächsten Seite, kannst du, falls nötig, noch letzte Anpassungen an deinen Pull-Request vornehmen. 
+Klicke erneut auf die Schaltfläche »Create pull request«.
+
+![Pull-Request erzeugen](/de/contributing/images/de/create-pull-request.png?classes=shadow)
+
+**Herzlichen Glückwunsch!** Du hast soeben einen Pull-Request für die offizielle Contao-Dokumentation erstellt! Die 
+Community wird nun deinen Pull-Request überprüfen und (möglicherweise) Änderungen vorschlagen.
+
+
+## Einen Beitrag für die Dokumentation schreiben
+
+Wenn du einen umfangreichen Beitrag planst oder du es vorziehst, an deinem eigenen Computer zu arbeiten, lese hier weiter 
+und wir zeigen dir eine alternative Möglichkeit auf, wie du Pull-Request an die Contao-Dokumentation senden kannst.
+
+### Git installieren
+
+Dazu musst du [Git auf deinem Betriebsystem installieren](https://git-scm.com/book/de/v2/Erste-Schritte-Git-installieren).
+
+
+### Fork des Repositories erstellen
+
+Wenn du noch kein Konto bei GitHub hast, [erstelle](https://github.com/join) eines und wechsle zum 
+offiziellen Contao-Dokumentations-Repository unter [github.com/contao/docs](https://github.com/contao/docs). Klicke auf 
+die Schaltfläche »[Fork](https://help.github.com/en/github/getting-started-with-github/fork-a-repo)«, um das Repository 
+auf dein persönliches Konto zu übertragen. Dies ist nur erforderlich, wenn du zum ersten Mal etwas zur Dokumentation 
+beiträgst.
+
+![Fork this repository](/de/contributing/images/de/fork-this-repository.png?classes=shadow)
+
+
+### Klone das geforkte Repository 
+
+Klone das geforkte Repository auf deinem lokalen Rechner:
+
+Erstelle ein Verzeichnis mit dem Namen `contao` und wechsle mit **c**hange **d**irectory  in dieses.
+
+```bash
+cd contao
+```
+
+Beim Klonen installierst du das Hugo Learn Theme als Submodul von git.
+
+```bash
+git clone --recurse-submodules git@github.com:DEIN-GITHUB-BENUTZERNAME/docs.git
+```
+
+oder
+
+```bash
+git clone --recurse-submodules https://github.com/DEIN-GITHUB-BENUTZERNAME/docs.git
+```
+
+Wechsle nachdem Klonen in das Verzeichnis »docs« und aktualisiere mit folgendem Befehl noch das Theme.
+
+```bash
+cd docs
+git submodule foreach git pull origin master
+```
```

Comment:
We should remove that. It might update the theme locally and then not match the state of the repo. 
(As commented in [Slack by Fritz](https://contao.slack.com/archives/GG9AHAH24/p1615970943006900)).

Oder, wenn dies beabsichtigt ist, mit entsprechenden Angaben. Fritz sagte dazu:

_... wenn das Theme der Docs noch nicht aktualisiert wurde. Du kannst natürlich einen PR bereitstellen wo du das Theme aktualisierst und gleichzeitig die neuen Features schon benutzt._

Reply:
The process of updating the theme should be reserved to the repository maintainers (i.e. @netzarbeiter and myself).